prover/t/Logic.v.expected

```diff
@@ -0,0 +1,84 @@
+<prover>
+  <k>
+    .CoqSentences
+  </k>
+  <exit-code>
+    1
+  </exit-code>
+  <goals>
+    .GoalCellSet
+  </goals>
+  <declarations>
+    <declaration>
+      axiom \equals ( AndComm , \lambda { A { Term } , B { Term } , H { Term } , .Patterns } \or ( \exists { H0 , H1 , .Patterns } \and ( \equals ( H , conj ( H0 , H1 , .Patterns ) ) , conj ( H1 , H0 , .Patterns ) , .Patterns ) , .Patterns ) )
```

Comment:
In the current master, axiom has always names attached: `axiom ax0 : \equals[...]`. Maybe just regenerate the file?

Reply:
Fixed, but should we remove the ability to have non-named axioms if we want to make named axioms standard?

Reply:
Maybe we should remove it. I will create an issue.

Reply:
Ad named axioms: you forgot to update the test files (`*.expected`).